lib/Constants.php

```diff
@@ -28,10 +28,16 @@
 const EBOOKS_DIST_PATH =		WEB_ROOT . '/ebooks/';
 const COVER_ART_UPLOAD_PATH =		'/images/cover-uploads/';
 
+const EBOOKS_IDENTIFIER_PREFIX =	'url:https://standardebooks.org/ebooks/';
+
 const DATABASE_DEFAULT_DATABASE = 	'se';
 const DATABASE_DEFAULT_HOST = 		'localhost';
 
 const EBOOKS_PER_PAGE = 12;
+const EBOOKS_MAX_STRING_LENGTH = 250;
+const EBOOKS_MAX_LONG_STRING_LENGTH = 500;
+const EBOOK_SINGLE_PAGE_SIZE_WARNING = 3 *1024 * 1024; // 3145728
+const EBOOK_EARLIEST_CREATION_DATE = new DateTimeImmutable('2014-01-01');
```

Comment:
I don't think we need to have this constant or check against it, seems strange to hard code the earliest ebook date.

Reply:
Done. I thought it would be useful to check for garbage dates that were way too early, e.g., 1970-01-01, but that hasn't been a problem so far.